### PR TITLE
Tables filtering

### PR DESCRIPTION
**Pull request by MariaDolotova**

### CODE REVIEW

File: UI/src/app/services/configuration.service.ts [hunk 1]
```diff
@@ -39,11 +39,16 @@ export class ConfigurationService {
     const newConfiguration = new Configuration({
       name: configurationName,
       mappingsConfiguration: this.bridgeService.arrowsCache,
-      tablesConfiguration: this.storeService.state.targetConfig
+      tablesConfiguration: this.storeService.state.targetConfig,
```
Comment (etokareva) [inline UI/src/app/services/configuration.service.ts:42]: move this.service.state to variable

File: UI/src/app/services/bridge.service.ts [hunk 2]
```diff
@@ -195,17 +201,17 @@ export class BridgeService {
   deleteArrowsForMapping(targetTableName: string, sourceTableName: string) {
     Object.keys(this.arrowsCache).forEach(key => {
       if (
-        this.arrowsCache[key].target.tableName.toUpperCase() === targetTableName.toUpperCase() &&
-        this.arrowsCache[key].source.tableName.toUpperCase() === sourceTableName.toUpperCase()
+        this.arrowsCache[ key ].target.tableName.toUpperCase() === targetTableName.toUpperCase() &&
```
Comment (etokareva) [inline UI/src/app/services/bridge.service.ts:204]: Let's use object destructuring with renaming

```ts
const cache = this.arrowsCache[ key ];
const { target: { tableName: cachedTargetTableName }, source: { tableName: cachedSourceTableName } }
```

File: UI/src/app/services/bridge.service.ts [hunk 3]
```diff
@@ -92,15 +92,21 @@ export class BridgeService {
     this.deleteAllArrows();
 
     this.arrowsCache = Object.assign(configuration.arrows);
+    this.storeService.add('filtered', configuration.filtered);
```
Comment (etokareva) [inline UI/src/app/services/bridge.service.ts:95]: We have setter in StoreService

```ts
this.storeService.state = { ...configuration }
```

File: UI/src/app/services/bridge.service.ts (hunk 2 above)
Comment (MariaDolotova) [reply UI/src/app/services/bridge.service.ts]: Done

**Pull request merged**